srcpkgs/nullmailer/INSTALL

```diff
@@ -0,0 +1,17 @@
+# INSTALL
+case "${ACTION}" in
+post)
+	if [ "${UPDATE}" = "no" ]; then
+		mkfifo -m 600 var/spool/nullmailer/trigger
+		chown _nullmail var/spool/nullmailer/*
+
+		chown _nullmail usr/bin/nullmailer-queue
+		chmod u+s usr/bin/nullmailer-queue
+		chown _nullmail usr/bin/mailq
```

Comment:
/usr/bin/mailq is also claimed by other mail packages which use symlinks.

Create a symlink of /usr/bin/nullmail to /usr/bin/mailq

same apply to sendmail

Reply:
You can look at `dma` `postfix` and `opensmtpd` for examples

Reply:
`nullmailer` provides a `sendmail` (1) man page.  Should this also be symlinked from, say `nullmailer-sendmail.1`, to `sendmail.1`, or shall I leave it as it is?